packages/utilities/fast-web-utilities/src/index.ts

```diff
@@ -8,3 +8,4 @@ export * from "./numbers";
 export * from "./strings";
 export * from "./query";
 export * from "./rtl-scroll-converter";
+export * from "./system-colors";
```

Comment:
seems like these will fail until you edit `src/index.ts` to `export * from "./system-colors";`